pyproject.toml

```diff
@@ -41,7 +41,8 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
-forking-recommended = ["ujson"]
+forking-recommended = ["ujson>=5.10.0"]
+etherscan-caching = ["requests-cache>=1.2.1"]
```

Comment:
I think these can be merged

Reply:
Just `recommended`?

Reply:
I guess it doesn't matter much, i was just thinking etherscan caching is mainly used with forking